### PR TITLE
update the code of rasterize to be compatible with the new version of selenium

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-7694?filter=62403)

## Description
Native image does not support chromium docker image anymore because of the selenium new version that had breaking changes in the service_args parameter.
This PR is to fix the code to be compatible with the new selenium version allowing us to update the chromium docker image.

### CODE REVIEW

File: Packs/rasterize/Integrations/rasterize/rasterize.yml [hunk 1]
```diff
@@ -311,7 +311,7 @@ script:
     - contextPath: InfoFile.Type
       description: The type of the image/pdf file
       type: string
-  dockerimage: demisto/chromium:1.0.0.56296
+  dockerimage: demisto/chromium:115.0.5790.69036
```
Comment (DeanArbel) [inline Packs/rasterize/Integrations/rasterize/rasterize.yml:314]: Is this right?

**Review by DeanArbel — APPROVED** [review]

File: Packs/rasterize/ReleaseNotes/1_2_23.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Fixed an issue where the service_args argument was not supported in the latest version of Selenium WebDriver.
```
Comment (DeanArbel) [inline Packs/rasterize/ReleaseNotes/1_2_23.md:6]:
```suggestion
- Fixed an issue where the *service_args* argument was not supported in the latest version of Selenium WebDriver.
```

File: Packs/rasterize/Integrations/rasterize/rasterize.yml (hunk 1 above)
Comment (omerKarkKatz) [reply Packs/rasterize/Integrations/rasterize/rasterize.yml:314]: I think there is a script that @JudahSchwartz wrote that changes the docker image name. please confirm

Comment (JudahSchwartz) [reply Packs/rasterize/Integrations/rasterize/rasterize.yml:314]: confirmed. 115.0.5790 is the chrome version

**Comment by omerKarkKatz** [comment]
Errors: Detect & Manage Phishing Campaigns - Test (Mock Disabled) - Extract Indicators From File - Generic v2 - Test (Mock Disabled) - Rasterize Test (Mock Disabled)

More info is in the on-call sheets.

**Pull request merged**